src/test/resources/test-scripts/organization.sql

```diff
@@ -24,6 +24,6 @@ INSERT INTO core.plan (identifier, json, date_deleted, server_version, id) VALUE
 INSERT INTO core.plan (identifier, json, date_deleted, server_version, id) VALUES ('7f2ae03f-9569-5535-918c-9d976b3ae5f8', '{}', null, 1567765433007, 294);
 
 
-INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (1, 1, 2243, 162, '2019-09-10 17:29:55.059000', '2022-09-10 17:29:55.059000',daterange('2019-09-10','2022-09-10'));
-INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (2, 1, 2243, 11, '2019-09-10 17:29:55.059000', '2022-09-10 17:29:55.059000',daterange('2019-09-10','2022-09-10'));
-INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (3, 2, 2243, 294, '2019-09-10 17:29:55.059000', '2022-09-10 17:29:55.059000',daterange('2019-09-10','2022-09-10'));
+INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (1, 1, 2243, 162, '2019-09-10 17:29:55.059000', '2023-09-10 17:29:55.059000',daterange('2019-09-10','2023-09-10'));
+INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (2, 1, 2243, 11, '2019-09-10 17:29:55.059000', '2023-09-10 17:29:55.059000',daterange('2019-09-10','2023-09-10'));
+INSERT INTO team.organization_location (id, organization_id, location_id, plan_id, from_date, to_date,duration) VALUES (3, 2, 2243, 294, '2019-09-10 17:29:55.059000', '2023-09-10 17:29:55.059000',daterange('2019-09-10','2023-09-10'));
```

Comment:
Change this to a later date so that the issues does not come up again soon before it's fixed and create an issue for this. This can possibly be handled in CHT